scripts/apps/web-publisher/index.js

```diff
@@ -0,0 +1,35 @@
+import './styles/web-publisher.scss';
+
+import { WebPublisherManagerController } from './controllers';
+import * as services from './services';
+
+export default angular.module('superdesk.apps.web_publisher', [
```

Comment:
You need to document this module too, see the [`authoring`](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/authoring/authoring/index.js#L12-L18) module as an example on how to do it. If you don't document the module, it's components will not be displayed (controllers, directives, etc) on the documentation page. It works as a hierarchy.